core/chains/evm/config/chain_specific_config.go

```diff
@@ -267,6 +267,15 @@ func setChainSpecificConfigDefaultSets() {
 	avalancheFuji := avalancheMainnet
 	avalancheFuji.linkContractAddress = "0x0b9d5D9136855f6FEc3c0993feE6E9CE8a297846"
 
+	// Harmony
+	harmonyMainnet := fallbackDefaultSet
+	harmonyMainnet.linkContractAddress = "0x218532a12a389a4a92fC0C5Fb22901D1c19198aA"
+	harmonyMainnet.gasPriceDefault = *big.NewInt(5000000000)
+	harmonyMainnet.minIncomingConfirmations = 3
```

Comment:
What are the finality rules on harmony? Is it probabilistic?

Reply:
Good question! 

No it's not probabilistic, it requires 2/3 majority quorum on a block for finality. According to them it's finality in 2 seconds. 
https://docs.harmony.one/home/general/technology/transactions

Do you think we could lower the `minIncomingConfirmations` ?

Reply:
It's not about time it's about how many blocks. How many blocks until finality on Harmony?

Reply:
according to them 1 block finality
![image](https://user-images.githubusercontent.com/50029043/134725560-59d4d325-4e94-4580-a86b-6818d8d362d6.png)

Reply:
Then, this can safely be set to 1

Reply:
Will do! Can y'all take a look again on Monday to see if there's any additional changes you recommend?